docs/examples/dodge-the-burgers.md

```diff
@@ -0,0 +1,59 @@
+# Dodge the Burgers
+
+Dodge the burgers and eat the fruits!
+
+```blocks
+enum SpriteKind {
+    Player,
+    Projectile,
+    Food,
+    Enemy
+}
+let projectile: Sprite = null
+let otherSprite: Sprite = null
+let sprite: Sprite = null
+sprites.onOverlap(SpriteKind.Player, SpriteKind.Projectile, function (sprite, otherSprite) {
+    info.changeLifeBy(-1)
+    scene.cameraShake(4, 500)
+    otherSprite.destroy(effects.bubbles)
+})
+sprites.onOverlap(SpriteKind.Player, SpriteKind.Food, function (sprite, otherSprite) {
+    info.changeScoreBy(1)
+    otherSprite.destroy()
+    sprite.startEffect(effects.hearts, 100)
+    music.baDing.play()
+})
+scene.setBackgroundColor(13)
+sprite = sprites.create(img`
```

Comment:
`sprite` should probably be `mySprite` to match the default and not conflict with overlap params